examples/execution-role.yaml

```diff
@@ -13,7 +13,9 @@ Resources:
         Statement:
           - Effect: Allow
             Principal:
-              Service: resources.cloudformation.amazonaws.com
+              Service:
+                - cloudformation.amazonaws.com
```

Comment:
[flyby] Caught me out when trying to use the execution-role created ARN role to deploy a resource, just failed saying the arn is invalid

Reply:
@Adelmar92 can you take a look ^^